src/components/pages/LoginPage.vue

```diff
@@ -0,0 +1,172 @@
+<script>
+import { defineComponent } from 'vue';
+import TheContainer from '@/components/common/TheContainer.vue';
+import AppInput from '@/components/common/App/AppInput.vue';
+import AppButton from '@/components/common/App/AppButton.vue';
+import FormCard from '@/components/views/FormCard.vue';
+import SectionTitle from '@/components/common/Section/SectionTitle.vue';
+import SectionSubtitle from '@/components/common/Section/SectionSubtitle.vue';
+import ValidationMessage from '@/components/common/Form/ValidationMessage.vue';
+import { useAuthStore } from '@/stores/authStore';
+
+export default defineComponent({
+  components: {
+    TheContainer,
+    AppInput,
+    AppButton,
+    FormCard,
+    SectionSubtitle,
+    SectionTitle,
+    ValidationMessage
+  },
+  setup() {
+    const auth = useAuthStore();
+    return { auth };
+  },
+  data() {
+    return {
+      email: '',
+      password: '',
+      emailTouched: false,
+      passwordTouched: false,
+    }
+  },
+  computed: {
+    isLoggedIn() {
+      return this.auth.isLoggedIn
+    },
+    emailErrorMessage() {
+      if (this.email === '' && this.emailTouched) {
+        return 'Please enter your email'
+      }
+      return ''
+    },
+    passwordErrorMessage() {
+      if (this.password === '' && this.passwordTouched) {
+        return 'Please enter your password'
+      }
+      return ''
+    },
+    isFormValid() {
+      return this.emailProvided && this.passwordProvided
+    }
+  },
+  methods: {
+    allTouched() {
+      this.emailTouched = true;
+      this.passwordTouched = true;
+    },
+    async handleLogin(e) {
+      e.preventDefault();
+      this.allTouched();
+      try {
+        await this.auth.login({
+          email: this.email,
+          password: this.password
+        })
+        this.$router.push({name: 'ChooseSeats'})
+      } catch (error) {
+        console.error(error)
+      }
+    }
+  }
+})
+</script>
+
+<template>
+  <div class="login-page">
+    <TheContainer variant="form">
+      <div class="login-page__header">
+        <SectionTitle size="large">Hi there!</SectionTitle>
+        <SectionSubtitle size="large">Care to log in?</SectionSubtitle>
+      </div>
+      <FormCard>
+        <form
+          @submit="handleSubmit"
```

Comment:
You don't have such method defined anywhere (also you should prevent it) 🤔 
```suggestion
          @submit.prevent="handleSubmit"
```

Reply:
👇